src/store/enhancedImageBuilderApi.ts

```diff
@@ -16,10 +16,7 @@ const enhancedApi = imageBuilderApi.enhanceEndpoints({
       },
     },
     cloneCompose: {
-      onQueryStarted: async (
-        { composeId, cloneRequest },
-        { dispatch, queryFulfilled }
-      ) => {
+      onQueryStarted: async ({ composeId }, { dispatch, queryFulfilled }) => {
```

Comment:
```suggestion
      onQueryStarted: async (
        { composeId, cloneRequest },
        { dispatch, queryFulfilled }
      ) => {
```

Something might have changed after all the rebases - now it seems the `cloneRequest` needs to be here.